packages/main/src/Breadcrumbs.ts

```diff
@@ -450,20 +503,48 @@ class Breadcrumbs extends UI5Element {
 		return text;
 	}
 
+	getCurrentLocationLabelWrapper() {
+		return this.shadowRoot!.querySelector<HTMLElement>(".ui5-breadcrumbs-current-location > span");
+	}
+
 	get _visibleItems() {
 		return this._getItems()
 			.slice(this._overflowSize)
 			.filter(i => this._isItemVisible(i));
 	}
 
-	get _endsWithCurrentPageItem() {
+	get _endsWithCurrentLinkItem() {
```

Comment:
minor: as the function name suggests it returns a boolean, maybe its result should be converted to boolean before return